pkg/gitops/utils.go

```diff
@@ -15,15 +15,52 @@
 package gitops
 
 import (
-	"path/filepath"
-
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"strings"
+    "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 
+    auditv1 "k8s.io/apiserver/pkg/apis/audit/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var dirMap = map[string]string{
+	"networkpoliciesnetworking.k8s.io":    "k8s-policies",
+	"networkpoliciescrd.antrea.io":        "antrea-policies",
+	"clusternetworkpoliciescrd.antrea.io": "antrea-cluster-policies",
+	"tierscrd.antrea.io":                  "antrea-tiers",
+}
+
+var resourceMap = map[string]string{
+	"networkpoliciesnetworking.k8s.io":    "K8s network policy ",
+	"networkpoliciescrd.antrea.io":        "Antrea network policy ",
+	"clusternetworkpoliciescrd.antrea.io": "Antrea cluster network policy ",
+	"tierscrd.antrea.io":                  "Antrea tier ",
+}
+
 func computePath(dir string, resource string, namespace string, file string) string {
-	return filepath.Join(dir, resource, namespace, file)
+	path := []string{}
+	for _, part := range []string{dir, resource, namespace, file} {
+		if part != "" {
+			path = append(path, part)
+		}
+	}
+	return strings.Join(path, "/")
+}
```

Comment:
filepath.Join() takes care of all this logic in computePath, so you should revert this change

Reply:
Noted, I also need to use filepath.Join() in other areas of the code as well.